src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -327,25 +325,7 @@ const formConfig = {
           title: 'Add a new disability',
           path: DISABILITY_SHARED_CONFIG.addDisabilities.path,
           depends: formData =>
-            DISABILITY_SHARED_CONFIG.addDisabilities.depends(formData) &&
-            !showRevisedNewDisabilitiesPage(),
-          uiSchema: addDisabilities.uiSchema,
-          schema: addDisabilities.schema,
-          updateFormData: addDisabilities.updateFormData,
-          appStateSelector: state => ({
-            // needed for validateDisabilityName to work properly on the review
-            // & submit page. Validation functions are provided the pageData and
-            // not the formData on the review & submit page. For more details
-            // see https://dsva.slack.com/archives/CBU0KDSB1/p1614182869206900
-            newDisabilities: state.form?.data?.newDisabilities || [],
-          }),
-        },
-        addDisabilitiesRevised: {
```

Comment:
Would renaming the page object impact the save-in-progress behavior you tested [here](https://dsva.slack.com/archives/C04AZ8T7XN1/p1722618205741999?thread_ts=1722617103.659859&cid=C04AZ8T7XN1)?

Also - will all the data still be available from saved in-progress forms for both `new-disabilities-revised/add` and `new-disabilities/add` users?

Reply:
I tested this as best I could in localhost and do not think it would redirect the user back to the first page in the form. I also did not notice any loss of data after changing the form object name. Each time i saved the form and changed the name of the form page, the data was still saved on the page.